builder/image/templates/builder

```diff
@@ -56,7 +56,12 @@ mkdir -p $BUILD_DIR $CACHE_DIR
 TMP_DIR=$(mktemp -d --tmpdir=$BUILD_DIR)
 
 cd $REPO_DIR
-git archive $GIT_SHA | tar -xmC $TMP_DIR
+git clone $REPO_DIR/ "$TMP_DIR/"
+pushd "$TMP_DIR"
```

Comment:
on line 67 we `cd` to $TMP_DIR. It'd be cleaner if we moved these operations below that line and removed the call to `cd $REPO_DIR` on line 58.